drivers/flash/flash_npcx_fiu_nor.c

```diff
@@ -253,12 +261,26 @@ static int flash_npcx_nor_read(const struct device *dev, off_t addr,
 	return 0;
 }
 
-static int flash_npcx_nor_erase(const struct device *dev, off_t addr,
-				  size_t size)
+static void disable_multiple_erase_sizes(void)
+{
+#undef ERASE_BLOCK_64K
```

Comment:
these lines always undefine ERASE_BLOCK_64K, ERASE_BLOCK_4K

Reply:
I removed macros at runtime in new revision, thank you